data-prepper-plugins/ruby-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/ruby/RubyProcessorIT.java

```diff
@@ -0,0 +1,63 @@
+package org.opensearch.dataprepper.plugins.processor.ruby;
+
+import org.checkerframework.checker.units.qual.C;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.record.Record;
+import org.opensearch.dataprepper.plugins.source.loggenerator.ApacheLogFaker;
+import org.opensearch.dataprepper.plugins.source.loggenerator.LogGeneratorSource;
+import org.opensearch.dataprepper.plugins.source.loggenerator.logtypes.CommonApacheLogTypeGenerator;
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.opensearch.dataprepper.test.helper.ReflectivelySetField.setField;
+
+public class RubyProcessorIT {
+    private static final int NUMBER_EVENTS_TO_TEST = 100;
+    private static final String PLUGIN_NAME = "ruby";
+    private static final String TEST_PIPELINE_NAME = "ruby_processor_test";
+    static String CODE_STRING = "require 'java'\n" +
```

Comment:
Copied from slack:


Having to require the customer have the to_java  is a bad customer experience. Ideally the Ruby code block should just be pure ruby without any jruby type conversions in there, and the Ruby processor should handle the translation between Ruby and Java behind the scenes.